cmd/autogen.sh

```diff
@@ -45,6 +45,9 @@ case "$ID" in
 		# patches find their way into the distribution.
 		extra_opts="--libexecdir=/usr/lib/snapd --disable-apparmor"
 		;;
+	solus)
+		extra_opts="--with-snap-mount-dir=/snap --enable-merged-usr --enable-nvidia-arch $@"
```

Comment:
In the Solus package we do:

`%autogen` which actually calls `./autogen.sh %CONFOPTS%` which passes all the configure arguments along, this makes `autogen.sh` complete the chain and pass them along to `configure` so that the whole thing is configured properly in one go